modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -812,31 +774,160 @@ values:
-|`missingValues`| string | nulls | How missing values are represented in the input data:
-
-* *none*: treat all data as actual values
-* *spaces*: interpret a field consisting of only spaces as a missing value
-* *zeroes*: interpret numeric fields consisting of only '0' characters and character fields consisting of only spaces as missing values
-* *nulls*: interpret a field consisting only of 0 bytes as a missing value
-
-| `recordParsing` | string | strict a| expected separation between lines/records:
-
-* *strict*: line break expected at exact end of each record
-* *lenient*: line break used but records may be shorter or longer than schema specifies
-* *noTerminator*: means records follow one another with no separation
+| Parameter | Type | Default | Description
+| `enforceRequires` | Boolean | `false` a| Error if required value missing.
+  Valid options: `true` or `false`. _Mule 3 support_: Version 3.9.2 and later.
+| `missingValues` | String | `nulls` for copybook schema, `spaces` otherwise a|
+
+Fill character used to represent missing values. To represent missing values
+in the input data, you can use:
+
+* `none`: Treat all data as actual values
+* `spaces`: Interpret a field consisting of only spaces as a missing value
+* `zeroes`: Interpret numeric fields consisting of only '0' characters and
+character fields consisting of only spaces as missing values
+* `nulls`: Interpret a field consisting only of 0 bytes as a missing value
+
+| `recordParsing` | String | `strict` a|
+
+Expected separation between lines/records:
+
+* `strict`: line break expected at exact end of each record
+* `lenient`: line break used but records may be shorter or longer than schema specifies
+* `noTerminator`: records follow one another with no separation
+* `singleRecord`: entire input is a single record (_Mule 3 support_: Version 3.9.2 and
+  later)
+
+| `schemaPath` | String | None a| Schema definition. Location in your local disk
+of the schema file used to parse your input.
+| `segmentIdent` | String | None a| Segment identifier in the schema for fixed
+width or copybook schemas (only needed when parsing a single segment/record
+  definition and if the schema includes multiple segment definitions).
+| `structureIdent` | String | None a| Structure identifier in schema for
+flat file schemas (only needed when parsing a structure definition, and if the
+schema includes multiple structure definitions).
+| `truncateDependingOn` | Boolean | `false` a| Truncate COBOL
+copybook DEPENDING ON values to length used. Valid options: `true` or `false`.
+_Mule 3 support_: Version 3.9.2 and later.
+| `zonedDecimalStrict` | Boolean | `false` a| Use the `strict` ASCII form of
+sign encoding for COBOL copybook zoned decimal values.
+Valid options: `true` or `false`. _Mule 3 support_: Version 3.9.2 and later.
 |===
 
+Note that schemas with type `Binary` or `Packed` don't allow for line break
```

Comment:
The "Not that" text is not necessary here and elsewhere. It just adds words.

Reply:
leaving it in